python_ta/debug/snapshot_tracer.py

```diff
@@ -17,20 +22,29 @@ class SnapshotTracer:
 
     Instance attributes:
         output_directory: The directory where the memory model diagrams will be saved. Defaults to the current directory.
+        open_webstepper: Opens the web-based visualizer.
+        _snapshot_to_line: A list of dictionaries that maps the code line number and the snapshot number.
```

Comment:
The description is inconsistent with the type annotation. But also, it seems the correct attribute name is `_snapshots`